package.json

```diff
@@ -94,20 +93,22 @@
   "dependencies": {
     "clsx": "^1.1.1",
     "lodash": "^4.17.20",
+    "nebenan-eventproxy": "^1.0.0",
     "nebenan-helpers": "^7.0.0",
+    "nebenan-keymanager": "^1.0.1",
     "nebenan-react-datepicker": "^2.0.0",
     "string-validate": "^3.5.0"
   },
   "peerDependencies": {
     "@babel/runtime": "^7.12.5",
     "nebenan-eventproxy": "^1.0.0",
     "nebenan-keymanager": "^1.0.1",
-    "nebenan-ui-kit": "^5.3.0",
+    "nebenan-ui-kit": "^5.4.0",
     "node-sass-glob-importer": "^5.3.2",
     "prop-types": "^15.7.2",
     "react": "^16.14.0 || ^17.0.1",
     "react-dom": "^16.14.0 || ^17.0.1",
     "sass": "^1.34.1",
-    "sass-functions": "^2.0.0 || ^1.4.0"
```

Comment:
Changing peerDependencies cause breaking releases. Don't think we need to update them here since we're not using any new ui-kit features - same thing with sass-functions